exercises/poker/src/example/java/Poker.java

```diff
@@ -0,0 +1,137 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class Poker {
+	private static class Card {
+		private int rank;
+		private int Suit;
```

Comment:
Variables normally start with a lowercase letter in java, so `suit` instead of `Suit` :)